test/build.sh

```diff
@@ -103,6 +103,10 @@ function build-ci() {
 
 function get-ci-env() {
   gsutil cp ${NPD_STAGING_PATH}/ci/${CI_ENV_FILENAME} ${ROOT_PATH}/${CI_ENV_FILENAME}
+
+  set -x
```

Comment:
Why `set -x` for the cat? Probably log the file name if that is what need here? `echo Using env file XXX`.

Reply:
Done